.github/workflows/haskell.yml

```diff
@@ -189,7 +190,7 @@ jobs:
       if: ${{ always() }}
       continue-on-error: true
       with:
-        name: chairman-test-artifacts-${{ matrix.os }}-${{ matrix.ghc }}
+        name: chairman-test-artifacts-${{ matrix.os }}-${{ matrix.n }}-${{ matrix.ghc }}
```

Comment:
This is needed so that when there is more than one `n`, the logs don't clobber each other.